Documentation/k8s-pre-reqs.md

```diff
@@ -1,6 +1,33 @@
 
-## New Local Kubernetes Cluster
+**WARNING:** Only Kubernetes v1.6.0 or higher is supported by Rook at this time (while Rook is in alpha it will track the latest release to use the latest features)
+
+## `kubeadm`
```

Comment:
Any reason you are adding quotes to the header title? it just look different than the rest of the headers. I would use "## Kubeadm" just to keep it consistent.